docker-compose.yml

```
@@ -9,15 +9,17 @@ services:
     volumes:
       - "beacon-chain-data:/data"
     ports:
-      - "12000/udp"
-      - "13000/tcp"
+      - "12004:12004/udp"
+      - "13004:13004/tcp"
```

```suggestion
      - "14003:14003/tcp"
```
This is the specified port in the port standardization.